health.go

```diff
@@ -325,9 +325,7 @@ func GetOSInfo(ctx context.Context, addr string) OSInfo {
 
 	osInfo.Sensors, err = host.SensorsTemperaturesWithContext(ctx)
 	if err != nil {
-		if _, isWarningErr := err.(*host.Warnings); !isWarningErr {
-			osInfo.Error = err.Error()
-		}
+		osInfo.Error = err.Error()
```

Comment:
We only need this error if its not host.Warnings() @mrusme

Reply:
To get this right, in case the following issue should arise, the error would not be needed?

https://github.com/shirou/gopsutil/blob/34cc43d282f68f93b2f9c5be19d0c4a6beb7d39d/host/host_linux.go#L386

In that case I guess the only option would be to remove the check altogether. What do you think?

Reply:
yes that is acceptable @mrusme